craft_application/models/manifest.py

```diff
@@ -0,0 +1,84 @@
+# This file is part of craft-application.
+#
+# Copyright 2024 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License version 3, as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+# SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Models representing manifests for projects and fetch-service assets."""
+from typing import Any, Literal
+
+from pydantic import Field
+from typing_extensions import override
+
+from craft_application.models import CraftBaseModel
+
+
+class Hashes(CraftBaseModel):
+    """Digests identifying an artifact/asset."""
+
+    sha1: str = Field(alias="SHA1")
```

Comment:
@cmatsuoka spec ST121 specifies "SHA-1" for the main component id but "SHA1" (different spelling) for the ids of the dependencies. Was this intentional? Here on the PR I just used the "SHA1" version for both

Reply:
Not intentional, for consistency with lowercase fields I'd say "sha1", "sha256", "sha384" would be the best choices.

Reply:
updated